### ISSUE TITLE
Unable to checkout Bundler repo on Windows

### ISSUE THREAD

**Issue by jakauppila** [bundler/bundler#5828]
It is currently not possible to checkout the Bundler repo on Windows due to the vcr_cassettes containing question marks in folder names added in #5220.

Would it be possible to convert the serialized request/response to another format?

```
D:\Dropbox\GIT_Workspaces>git clone https://github.com/bundler/bundler
Cloning into 'bundler'...
remote: Counting objects: 70934, done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 70934 (delta 18), reused 14 (delta 10), pack-reused 70906
Receiving objects: 100% (70934/70934), 31.52 MiB | 6.92 MiB/s, done.
Resolving deltas: 100% (45282/45282), done.
fatal: cannot create directory at 'spec/support/artifice/vcr_cassettes/realworld/api.rubygems.org/api/v1/dependencies?gems=bundler': Invalid argument
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

**Comment by colby-swandale** [comment]
\cc @segiddins

**Comment by segiddins** [comment]
Why is `?` in file names an issue? As far as I'm aware, it's not `File::SEPARATOR` on any platform?

**Comment by jakauppila** [comment]
The following are reserved characters on Windows:


```
The following reserved characters:
- < (less than)
- > (greater than)
- : (colon)
- " (double quote)
- / (forward slash)
- \ (backslash)
- | (vertical bar or pipe)
- ? (question mark)
- * (asterisk)
```
https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247

**Comment by colby-swandale** [comment]
fixed in #5829

**Comment by jakauppila** [comment]
Thanks! It is now possible to check out the repo.
